packages/data-stores/src/add-ons/types.ts

```diff
@@ -0,0 +1,20 @@
+import { TranslateResult } from 'i18n-calypso';
+
+export interface AddOnMeta {
+	productSlug: string;
+	featureSlugs?: string[] | null;
+	icon: JSX.Element;
+	featured?: boolean; // irrelevant to "featureSlugs"
```

Comment:
It's unclear to me what it means by "irrelevant to featureSlugs". It's been this way so I'm fine with us keeping it for a bit longer, but we should definitely update it into something more meaningful in the near future.

Reply:
I tracked down what it's used for and updated the inline comment https://github.com/Automattic/wp-calypso/pull/81338/commits/cc793eaede2b7d2eaa7ffc0e505303699ad766b4